packages/amplify_api_rest/android/src/test/kotlin/com/amazonaws/amplify/amplify_api_rest/AmplifyApiRestTest.kt

```diff
@@ -0,0 +1,414 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.amplify.amplify_datastore
+
+import com.amazonaws.amplify.amplify_api_rest.AmplifyApiRestPlugin
+import com.amplifyframework.api.ApiCategory
+import com.amplifyframework.api.ApiException
+import com.amplifyframework.api.aws.AWSApiPlugin
+import com.amplifyframework.api.rest.RestOperation
+import com.amplifyframework.api.rest.RestOptions
+import com.amplifyframework.api.rest.RestResponse
+
+import com.amplifyframework.core.Amplify
+import com.amplifyframework.core.Consumer
+import com.amplifyframework.hub.HubEvent
+import com.amplifyframework.hub.SubscriptionToken
+import io.flutter.plugin.common.MethodCall
+import io.flutter.plugin.common.MethodChannel
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+import org.mockito.Mockito.*
+import org.mockito.invocation.Invocation
+import org.mockito.invocation.InvocationOnMock
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.Shadows.shadowOf
+import java.lang.reflect.Field
+import java.lang.reflect.Modifier
+import java.util.concurrent.CountDownLatch
+
+@RunWith(RobolectricTestRunner::class)
+class AmplifyApiRestTest {
+    lateinit var flutterPlugin: AmplifyApiRestPlugin
+
+    private var mockApi = mock(ApiCategory::class.java)
+    private var mockAWSApiPlugin = mock(AWSApiPlugin::class.java)
+
+    private val mockResult: MethodChannel.Result = mock(MethodChannel.Result::class.java)
+    private val mockRestOperation: RestOperation = mock(RestOperation::class.java)
+
+    @Before
+    fun setup() {
+        flutterPlugin = AmplifyApiRestPlugin()
+
+        setFinalStatic(Amplify::class.java.getDeclaredField("API"), mockApi)
+    }
+
+    @Test
+    fun test_get_returns_success(){
+
+        var data = "{\"success\":\"get call succeed!\",\"url\":\"/items\"}".toByteArray()
+        var restResponse = RestResponse(200, data)
+
+        Mockito.doAnswer { invocation: InvocationOnMock ->
+            Assert.assertEquals(
+                    RestOptions.builder().addPath("/items").build(),
+                    invocation.arguments[0]
+            )
+            (invocation.arguments[1] as Consumer<RestResponse>).accept(
+                    restResponse
+            )
+            mockRestOperation
+        }.`when`(mockApi).get(
+                ArgumentMatchers.any(RestOptions::class.java),
```

Comment:
I like to use static imports for `any`, to cut down on the boiler plate. Mockito is written in a very declarative way that can read like a sentence if you use its method names directly.

Reply:
nice - removed "ArgumentMatchers." boilerplate code